fix-imports.sh

```
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+# Fix @/components/ui imports
+find src -name "*.tsx" -o -name "*.ts" | xargs sed -i "s|from '@/components/ui/|from './ui/|g"
```

Shellcheck (0.11.0)

[warning] Use 'find .. -print0 | xargs -0 ..' or 'find .. -exec .. +' to allow non-alphanumeric filenames.

(SC2038)